### PR TITLE
Use ChatterBot for Sesheta

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
Sadly, there is a collision in versions:

```
  pyyaml<4.0,>=3.12 (from chatterbot-corpus==1.2.0->-r /tmp/pipenv9f1i85zfrequirements/pipenv-c4_eis7l-constraints.txt (line 10))
  pyyaml<5.2,>=5.1 (from chatterbot==1.0.5->-r /tmp/pipenv9f1i85zfrequirements/pipenv-c4_eis7l-constraints.txt (line 12))
```

ChatterBot in version 1.1.0 with a fix for this was yanked. Mareover I cannot install the git version as:

https://github.com/gunthercox/ChatterBot/blob/d911b4ac46e122308a759ea6ec67b666136e3026/setup.py#L72

produces:

```
ERROR: Package 'ChatterBot' requires a different Python: 3.8.3 not in '>=3.4, <=3.8'
```

**Comment by saisankargochhayat** [comment]
> Sadly, there is a collision in versions:
> 
> ```
>   pyyaml<4.0,>=3.12 (from chatterbot-corpus==1.2.0->-r /tmp/pipenv9f1i85zfrequirements/pipenv-c4_eis7l-constraints.txt (line 10))
>   pyyaml<5.2,>=5.1 (from chatterbot==1.0.5->-r /tmp/pipenv9f1i85zfrequirements/pipenv-c4_eis7l-constraints.txt (line 12))
> ```
> 
> ChatterBot in version 1.1.0 with a [fix for this](https://github.com/gunthercox/ChatterBot/commit/d911b4ac46e122308a759ea6ec67b666136e3026) was yanked. Mareover I cannot install the git version as:
> 
> https://github.com/gunthercox/ChatterBot/blob/d911b4ac46e122308a759ea6ec67b666136e3026/setup.py#L72
> 
> produces:
> 
> ```
> ERROR: Package 'ChatterBot' requires a different Python: 3.8.3 not in '>=3.4, <=3.8'
> ```

If upstream doesn't take it maybe we will do a fork, lets see if they accept it. :+1:

**Comment by fridex** [comment]
> > Sadly, there is a collision in versions:
> > ```
> >   pyyaml<4.0,>=3.12 (from chatterbot-corpus==1.2.0->-r /tmp/pipenv9f1i85zfrequirements/pipenv-c4_eis7l-constraints.txt (line 10))
> >   pyyaml<5.2,>=5.1 (from chatterbot==1.0.5->-r /tmp/pipenv9f1i85zfrequirements/pipenv-c4_eis7l-constraints.txt (line 12))
> > ```
> > 
> > 
> > ChatterBot in version 1.1.0 with a [fix for this](https://github.com/gunthercox/ChatterBot/commit/d911b4ac46e122308a759ea6ec67b666136e3026) was yanked. Mareover I cannot install the git version as:
> > https://github.com/gunthercox/ChatterBot/blob/d911b4ac46e122308a759ea6ec67b666136e3026/setup.py#L72
> > produces:
> > ```
> > ERROR: Package 'ChatterBot' requires a different Python: 3.8.3 not in '>=3.4, <=3.8'
> > ```
> 
> If upstream doesn't take it maybe we will do a fork, lets see if they accept it.

I added my fork in 7b14e02 in case we would like to give it a try. Dependencies will not be managed by kebechet though.

File: aicoe/sesheta/chatbot.py [hunk 1]
```diff
@@ -52,6 +52,9 @@
  "sub-mod",
  "xtuchyna",
 ]
+_CHATBOT = ChatBot("Sesheta")
```
Comment (fridex) [inline aicoe/sesheta/chatbot.py:55]: It mighe be good idea to have the bot in read-only mode - [see the docs](https://chatterbot.readthedocs.io/en/stable/examples.html#read-only-mode):

```suggestion
_CHATBOT = ChatBot("Sesheta", read_only=True)
```

Comment (saisankargochhayat) [reply aicoe/sesheta/chatbot.py:55]: We don't want it to learn? :stuck_out_tongue:

**Review by saisankargochhayat — APPROVED** [review]
Let's merge and try this :guitar:

File: aicoe/sesheta/chatbot.py (hunk 1 above)
Comment (fridex) [reply aicoe/sesheta/chatbot.py]: To my understanding the flag turns on "contextual" conversation which might be not the right thing for long-living bot in a channel.

**Pull request merged**